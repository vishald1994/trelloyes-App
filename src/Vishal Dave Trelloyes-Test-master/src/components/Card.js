import React from 'react';
import '../styles/Card.css';

class Card extends React.Component {
  render() {
    return (
      <div className="Card">
        <button type="button">delete</button>
        <h3>{this.props.title}</h3>
        <p>{this.props.content}</p>
      </div>
    )
  }
}

export default Card;