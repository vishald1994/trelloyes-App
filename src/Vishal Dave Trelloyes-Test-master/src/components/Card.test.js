import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Card from './Card';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Card
    key='0'
    title='Test Title'
    content='Test Content stufdisfjaoewijfeow'
    />, div);

  ReactDOM.unmountComponentAtNode(div);
})

it ('renders this UI as expected', () => {
  const app = renderer.create(<Card key='0' title='test title' content='test content'/>).toJSON();
  expect(app).toMatchSnapshot();
})

