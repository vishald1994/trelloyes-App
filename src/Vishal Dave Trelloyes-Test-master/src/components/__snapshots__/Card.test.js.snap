// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders this UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    test title
  </h3>
  <p>
    test content
  </p>
</div>
`;
